### PR TITLE
Updated the READMEs

### LINKED ISSUE #220: Document how to use this image on GKE

### PR DESCRIPTION
**Pull request by tmatsuo**
fixes #220

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Docker image for the App Engine Flexible PHP runtime
 
-This repository contains experimental PHP runtime(s) for Google App
-Engine Flexible Environment. It is not covered by any SLA or
-deprecation policy.  It may change at any time.
+This repository contains experimental PHP runtime for Google App
```
Comment (chingor13) [inline README.md]: contains **an** experimental PHP runtime

**Comment by tmatsuo** [comment]
@dpebot merge when green

**Comment by tmatsuo** [comment]
@dpebot merge when green

**Comment by tmatsuo** [comment]
@dpebot merge when green

**Comment by tmatsuo** [comment]
@jonparrott The dpebot seems lazy here. The webhook for the last test success was correctly sent to the webhook about 30 minutes ago, so I think something went wrong with him. I'll appreciate if you can take a look.

**Comment by tmatsuo** [comment]
@dpebot merge when green
(Let me try agian)

**Comment by tmatsuo** [comment]
@jonparrott Oh now it seems working, nm

**Comment by tmatsuo** [comment]
@dpebot 

Great job! Thanks

**Pull request merged**